Jenkinsfile

```diff
@@ -51,27 +51,57 @@ try {
             node('docker') {
                 deleteDir()
                 git 'https://github.com/jenkins-infra/azure.git'
+                /* Create an empty terraform variables file so that everything can
+                 * be overridden in the environment
+                 */
                 sh "echo '{\"prefix\":\"${tfPrefix}\"}' > ${tfVarFile}"
+
+                /*
+                 * Three following actions must be remove once terraform is greater than 0.8
+                 * Cfr INFRA-1288
```

Comment:
wtf does cfr mean :smiling_imp: 

But seriously, I don't understand why this is here, since this PR ugprades to a Terraform greater than 0.8